src/components/tx-flow/flows/AddOwner/ReviewOwner.tsx

```diff
@@ -57,8 +58,26 @@ export const ReviewOwner = ({ params }: { params: AddOwnerFlowProps | ReplaceOwn
       <Paper sx={{ backgroundColor: ({ palette }) => palette.background.main, p: '12px' }}>
         <EthHashInfo name={addressBook[safeAddress]} address={safeAddress} shortAddress={false} />
```

Comment:
The label ("Selected Safe Account") [is missing](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?type=design&node-id=7057-169722&mode=design&t=aZRJCFMNH1sCOvcR-0). I'd also check how this